internal/cmd/stack/open_command.go

```diff
@@ -110,22 +110,16 @@ func getGitRepositorySubdir() (string, error) {
 
 	root := current
 	for {
-		if _, err := os.Stat(".git"); err == nil {
+		if _, err := os.Stat(filepath.Join(root, ".git")); err == nil {
 			break
 		} else if !os.IsNotExist(err) {
 			return "", fmt.Errorf("couldn't stat .git directory: %w", err)
 		}
+		root = filepath.Dir(root)
+	}
 
-		cwd, err := os.Getwd()
-		if err != nil {
-			return "", fmt.Errorf("couldn't get current working directory: %w", err)
-		}
-
-		parent := filepath.Dir(cwd)
-		if err := os.Chdir(parent); err != nil {
-			return "", fmt.Errorf("couldn't set current working directory: %w", err)
-		}
-		root = parent
+	if !strings.HasSuffix(current, "/") {
```

Comment:
I don't know what the correct behavior is here.

Can we assume that the project root always has a trailing slash?

I don't think I have seen any guidance on this in the docs.